cpp/IceBox/hello/Client.cpp

```diff
@@ -53,20 +53,13 @@ void menu();
 int
 run(const shared_ptr<Ice::Communicator>& communicator)
 {
-    auto twoway =
-        Ice::checkedCast<HelloPrx>(communicator->propertyToProxy("Hello.Proxy")->ice_twoway()->ice_secure(false));
-    if (!twoway)
-    {
-        cerr << "invalid proxy" << endl;
-        return 1;
-    }
+    HelloPrx twoway{communicator, "hello:tcp -h localhost -p 10000:udp -h localhost -p 10000"};
```

Comment:
We should further simplify in a follow up PR and have IceBox/greeter, inline with `Ice/greeter`.

Reply:
and remove all udp support?